presto-docs/src/main/sphinx/release/release-0.140.rst

```diff
@@ -22,6 +22,12 @@ General Changes
   duplicated columns.
 * Optimize ``NOT IN`` queries to produce more compact predicates.
 * ``query.max-age`` property was renamed to ``query.min-expire-age``.
```

Comment:
We will be getting rid of the columnar_processing and only have the optimization for dictionary processing soon. So we don't need this change.